### ISSUE TITLE
Camera Mount

### ISSUE THREAD

**Issue by amcaleer271** [uf-mil-mechanical/tasks#26]
The current camera mount for the front facing camera on NaviGator is 3D printed and currently cracking. 

Requirements:
* The mount should maintain the current camera orientation
* The mount should be able to be attached to 80-20 aluminum extrusion using 1/4-20 hardware.

**Comment by amcaleer271** [comment]
Assigned to Ryan Stewart

**Comment by amcaleer271** [comment]
Assigned to Bryan

**Comment by BryanBoc** [comment]
![Image](https://github.com/user-attachments/assets/eb759241-9cad-4fbc-b305-ed4b573bcb32)
Here's what we have for the camera mount assembly right now. We took the tube from the old NaviGator camera and made a clamping mechanism to hold it in place inspired by the ones on Sub 9. We just need to order the tape to wrap around the inside which Ryan looked into. I talked to one of the software girls and she said 15 degrees downward would be good but we can always change that or anything else if needed.

**Comment by RyanSt3wart** [comment]
Next steps moving forward are to order the tape and start printing the 3Dp parts out of PLA to test if the tolerances are correct. We will also need to check if we have the correct M2 screws for the sled to mount onto the tube end caps and the 1/4-20s for the mounting.

**Comment by RyanSt3wart** [comment]
All printed parts have been printed and tolerances have been fixed on the sled to ensure accessibility and stability. We still need to order the grip tape and check on what screws we need to order once the metric screws are sorted by leads. We also need to talk with leads to discuss how to mount the camera mount on the 80-20 structure for the top of the boat.

**Comment by BryanBoc** [comment]
![Image](https://github.com/user-attachments/assets/dc336326-d9d6-4dc7-a310-96902a1abd15)

**Comment by amcaleer271** [comment]
These look good, once you have the design finalized find a place to mount them on the assembly.

**Comment by RyanSt3wart** [comment]
![Image](https://github.com/user-attachments/assets/13d7bf50-0143-4c45-9f15-9435b81d30b5)

Need to find correct M2 screws to attach polycarbonate cover and change mounting holes on bottom to fit with 80-20 structure on top of boat.

**Comment by BryanBoc** [comment]
^^. Also need to change tolerance of holes on reprint.

**Comment by RyanSt3wart** [comment]
Changed the mounting holes on the legs to line up with the 80-20 structure. Need to combine in CAD before 3D-Printing the final designs and may need to still change the tolerances of the holes.

**Comment by BryanBoc** [comment]
Finishing up CAD for final design which will fit onto the 80-20. Will check with one of the leads before printing. 

![Image](https://github.com/user-attachments/assets/897327a3-8391-4265-9f71-bcf3937fa3b6)

**Comment by BryanBoc** [comment]
Gonna try to get this fully printed this week

**Comment by RyanSt3wart** [comment]
Need to combine the new bottom parts of the legs in cad then do final print.

**Comment by RyanSt3wart** [comment]
Finished changing holes and configuring new legs in CAD. We are ready for the final print. We are waiting on PETG filament to arrive from Bambu

**Comment by RyanSt3wart** [comment]
Setup printer with new PETG from Bambu. Currently printing final updated prints.

**Comment by BryanBoc** [comment]
^ Gonna try to get it fully printed and assembled early next week.

**Comment by BryanBoc** [comment]
Still getting this printed... had some problems with the printer but I think we figured it out.

**Comment by RyanSt3wart** [comment]
^ This + Added additional supports to the slice

**Comment by BryanBoc** [comment]
fully printed, needs to be assembled and put onto NavigGator. Last time I was in Cimar the 80-20 was not the same as the fusion but I believe they're still working on it so I'll check it out this week

**Comment by RyanSt3wart** [comment]
Still need to drill the holes out to .265 for the 2 top camera mount clamps.

**Comment by amcaleer271** [comment]
It looks like these are basically done, just need a back on the tubes and to be mounted to the boat!

**Comment by adamhamdan8** [comment]
put it on the boat then we can close this

**Comment by RyanSt3wart** [comment]
Okay, we will mount it after we get back from break. Where did the boat get moved to?

**Comment by BryanBoc** [comment]
yup, got all the fasteners and stuff ready, and I believe Ryan ordered the cord for the camera

**Comment by amcaleer271** [comment]
With not using NaviGator for RobotX we don't need this anymore. We may be able to use the same/similar mount for cameras on the new boat though! Closing issue
